rdata/io/xdr.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
```

Comment:
It could indeed be a separate package, but it would require quite some extra work to have a full xdrlib replacement. The code here also requires numpy, so it wouldn't be a particularly nice general replacement of xdrlib.